polygon/rest/base.py

```diff
@@ -67,8 +67,9 @@ def _get(
 
         if result_key:
             obj = obj[result_key]
-        else:
-            # If the result_key does not exist, still need to put the results in a list
+
+        # If obj is not yet a list, need to turn it into a list
+        if type(obj) != list:
             obj = [obj]
```

Comment:
Why is this needed? I think it's okay if this method doesn't always return a list.

Reply:
last trade does not work without this

Reply:
Ah, I never expected "results" to not be a list. This is good.